share/picongpu/tests/Synchrotron_plugin/lib/validate.py

```diff
@@ -114,11 +134,12 @@ def main(dataPath):
     analytical_integrated = np.array(analytical_integrated)[mask]
 
     x, y = subtract_functions(delta, analytical_integrated, b, a)
-    poorness = np.sum(y) / len(y)  # average error
+    poorness = np.sum(y) / len(y)  # average relative error
```

Comment:
Use `np.mean` (potentially with `axis=0`) here.

Reply:
We left it because in the future we might do something like np.sum(y*np.sqrt(a)) / len(y)

Reply:
That's still a mean isn't it? `np.mean(y*np.sqrt(a))`

Reply:
I agree with @chillenzer

Reply:
and why is this now "relative" ?

Reply:
It was always relative